crates/cli/src/benchmark.rs

```diff
@@ -308,7 +314,7 @@ impl BenchmarkCommand {
             // and therefore potentially invalidating relative paths used here).
             let engine = check_engine_path(engine)?;
 
-            for wasm in &self.wasm_files {
+            for wasm in &self.benchmarks {
```

Comment:
Not sure this looks right: I feel like we should be pushing the actual benchmark paths into `choices`, not entire suites.

Reply:
What I ended up doing here is using `flat_map` so that each subprocess in `execute_in_multiple_processes` gets a single benchmark to run instead of potentially receiving a whole suite. `execute_in_current_process` stays the same in that it can execute either a suite or a single benchmark.

> We also have the `wasm_files` available in scope already (although they are `String`s and should probably be `PathBuf`s instead) that we can reuse for this loop.

We don't have `wasm_files` in this function `execute_in_multiple_processes`, only in `execute_in_current_process`. I did try, however, to see if I could use `PathBuf`s instead of `String`s but the fly in the ointment is `Measurements`: it wants a `&'a str` to refer to the Wasm file and I would guess `wasm_files` became a `Vec<String>` to satisfy that lifetime. There is probably a refactoring here to get to, e.g., `&Path` but I'll leave it as-is for this PR.